policy/scan/local_scanner.go

```diff
@@ -249,6 +249,7 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 			Asset:    resolvedAsset,
 			Upstream: upstream,
 		}); err != nil {
+			reporter.AddScanError(resolvedAsset, err)
```

Comment:
Is there a way, we can test this?

As these are connect errors, I'm not sure these will end up as intended here: https://github.com/mondoohq/cnspec/blob/5438e55fcc06668ea7c35466664d96b6c97e8ea6/policy/scan/aggregate_reporter.go#L46-L47

I worked at something similar today and had no assert information.

Reply:
that's a good point, we will end up with errs being overwritten here. need to go back to the drawing board on this one